### PR TITLE
[ASTGen] Split ASTBridging.cpp into multiple files

### PR DESCRIPTION
**Pull request by rintaro**
ASTBridging.cpp was getting too big.

### CODE REVIEW

**Comment by rintaro** [comment]
@swift-ci Please smoke test

File: lib/AST/Bridging/Pluigins.cpp
Comment (hamishknight) [inline lib/AST/Bridging/Pluigins.cpp:1]: Typo in filename

File: lib/AST/Bridging/Expr.cpp
Comment (hamishknight) [inline lib/AST/Bridging/Expr.cpp:1]: Empty file?

**Review by eeckstein — APPROVED** [review]

**Comment by rintaro** [comment]
@swift-ci Please smoke test

**Review by hamishknight — APPROVED** [review]

**Pull request merged**